drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -936,6 +936,7 @@ void mt76_connac_mcu_wtbl_ba_tlv(struct mt76_dev *dev, struct sk_buff *skb,
 		ba->rst_ba_tid = params->tid;
 		ba->rst_ba_sel = RST_BA_MAC_TID_MATCH;
 		ba->rst_ba_sb = 1;
+		ba->ba_winsize = enable ? cpu_to_le16(params->buf_size) : 0;
```

Comment:
@objelf since mt76_connac_mcu_wtbl_ba_tlv() is used by mt7615 and mt7663 I guess we should move this inside the if (is_mt7921()) check. I will fix it applying the patch